gitpod-io/gitpod
# [rebuild] support desktop IDEs

## Description


This PR enables support of VS Code Desktop and JB product in workspaces started by gp rebuild.

## Related Issue(s)

Fixes #

## How to test


- Download dev version of JB GW plugin associated with this branch fro JB Marketplace and install. in your JB GW.
- Ensures to upgrade Gitpod extension in VS Code Desktop to latest version. 
- Start a workspace with VS Code Desktop or IntelliJ.
- Confirm that usual flow works.
- Now start a workspace with gp rebuild and confirm that such flow works as well.

## Release Notes

```release-note
NONE
```

## Documentation


## Build Options:

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
